eng/scripts/Update-PkgVersion.ps1

```diff
@@ -40,7 +40,7 @@ function Update-Version(
     [AzureEngSemanticVersion]$SemVer,
     $VersionHppLocation,
     $Unreleased=$True,
-    $ReplaceVersion=$False)
+    $ReplaceLatestEntryTitle=$False)
```

Comment:
@chidozieononiwu looks like you missed updating the C++ repo. Can you please make sure there aren't any references in the other secondary language repos?